### PR TITLE
Added `name` and `symbol` to ERC20

### PR DESCRIPTION
**Pull request by andrew-fleming**
# Added `name` and `symbol` to ERC20

## Summary 

This PR, in reference to #59  , adds `name` and `symbol` functions to the ERC20 implementation. The two storage variables `_name` and `_symbol`, to which these functions fetch, are set in the constructor. Cairo, however, does not accept short string arguments because the compiler expects an `int` type; therefore, the tests include a helper function `str_to_felt()`. This helper function follows how Cairo interprets short string literals. If it's preferable to temporarily hardcode `_name` and `_symbol`, the helper function will be much more helpful in testing.

## Implementation
- Added `storage_vars` for `_name` and `_symbol`
- Added `view` functions `name()` and `symbol()`
- Inserted two more parameters in the ERC20 constructor
- Created helper function `str_to_felt()` for testing
- Set in "Token" and "TKN" as `_name` and `_symbol` arguments in testing constructor

### CODE REVIEW

File: contracts/token/ERC20.cairo [hunk 1]
```diff
@@ -12,6 +12,14 @@ from starkware.cairo.common.uint256 import (
 # Storage
 #
 
+@storage_var
+func _name() -> (res: felt):
```
Comment (martriay) [inline contracts/token/ERC20.cairo:16]: I think we can drop the underscore right? There's no collision risk.

File: contracts/token/ERC20.cairo [hunk 2]
```diff
@@ -37,9 +45,11 @@ func constructor{
         syscall_ptr : felt*, 
         pedersen_ptr : HashBuiltin*,
         range_check_ptr
-    }(recipient: felt):
+    }(name: felt, symbol: felt, recipient: felt):
```
Comment (martriay) [inline contracts/token/ERC20.cairo:48]:
```suggestion
    }(
        name: felt,
        symbol: felt,
        recipient: felt
    ):
```

File: tests/test_ERC20.py [hunk 3]
```diff
@@ -14,6 +14,11 @@ def uint(a):
     return(a, 0)
 
 
+def str_to_felt(text):
+    b_text = bytes(text, 'UTF-8')
+    return int.from_bytes(b_text, "big")
```
Comment (martriay) [inline tests/test_ERC20.py:19]: 👍

**Review by martriay — APPROVED** [review]

File: contracts/token/ERC20.cairo (hunk 1 above)
Comment (andrew-fleming) [reply contracts/token/ERC20.cairo:16]: The compiler doesn't like it. "Redefinition of name" because it's in `view` and `storage_var`

**Pull request merged**